include/astra_camera/ros12_shim.h

```diff
@@ -1,39 +1,20 @@
 #ifndef ROS12_SHIM_H_
 #define ROS12_SHIM_H_
 
-#include <stdio.h>
 #include <iostream>
 
-#ifndef ROS_INFO
-  #define ROS_INFO(str, ...) printf(str "\n", ## __VA_ARGS__)
-#endif
-#ifndef ROS_DEBUG
-  #define ROS_DEBUG(str, ...)
-#endif
-#ifndef ROS_WARN
-  #define ROS_WARN(str, ...) printf(str "\n", ## __VA_ARGS__)
-#endif
-#ifndef ROS_ERROR
-  #define ROS_ERROR(str, ...) printf(str "\n", ## __VA_ARGS__)
-#endif
-#ifndef ROS_FATAL
-  #define ROS_FATAL(str, ...) printf(str "\n", ## __VA_ARGS__)
-#endif
+#include <rcutils/logging_macros.h>
 
-#ifndef ROS_INFO_STREAM
-  #define ROS_INFO_STREAM(str) std::cout << str << std::endl
-#endif
-#ifndef ROS_DEBUG_STREAM
-  #define ROS_DEBUG_STREAM(str)
-#endif
-#ifndef ROS_WARN_STREAM
-  #define ROS_WARN_STREAM(str) std::cout << str << std::endl
-#endif
-#ifndef ROS_ERROR_STREAM
-  #define ROS_ERROR_STREAM(str) std::cout << str << std::endl
-#endif
-#ifndef ROS_FATAL_STREAM
-  #define ROS_FATAL_STREAM(str) std::cout << str << std::endl
-#endif
+#define ROS_INFO RCUTILS_LOG_INFO
+#define ROS_DEBUG RCUTILS_LOG_DEBUG
+#define ROS_WARN RCUTILS_LOG_WARN
+#define ROS_ERROR RCUTILS_LOG_ERROR
+#define ROS_FATAL RCUTILS_LOG_FATAL
+
+#define ROS_INFO_STREAM(str) std::cout << str << std::endl
+#define ROS_DEBUG_STREAM(str)
```

Comment:
I know it was that way before but would it make sense for debug messages to be streamed to cout as well?
Slightly related should the WARN severity level and higher be printed to cerr rather than cout?

Reply:
Yeah, I guess those should be cleaned up.  I'll do that.

Reply:
Actually, I went a little further and removed all of the shims we weren't using, and also changed the ERROR stream to point to cerr.